server/services/database/message.js

```diff
@@ -0,0 +1,21 @@
+const { Pool } = require("pg");
+const config = require("../../config");
+const pool = new Pool(config);
+
+const createMessage = ({ recruiter_email, profile_public_id, message }) => {
+  return pool.query(
+    "INSERT INTO message (recruiter_email, profile_public_id, message) values ($1, $2, $3)",
+    [recruiter_email, profile_public_id, message],
+  );
+};
+
+const getMessagesByApplicantId = (id) => {
+  return pool
+    .query("SELECT * FROM message WHERE applicant_id=$1", [id])
```

Comment:
there is no applicant_id in message

select  u.email recruiter_email, p.public_profile_id, m.message message from message m, profile p, users u where u.id=m.recruiter_id and p.id=m.profile_id and p.applicant_id=$1